tsconfig.json

```diff
@@ -1,17 +1,19 @@
 {
```

Comment:
Please help me exclude node_modules 🥰

Reply:
What do you wanna achieve? Just setting `skipLibsCheck` to `true`?

Reply:
I wanna speed up the compilation, but maybe lead to type riskies. So keep as it 😃